fission/spec.go

```diff
@@ -520,6 +520,12 @@ func (fr *FissionResources) parseYaml(b []byte, loc *location) error {
 // readSpecs reads all specs in the specified directory and returns a parsed set of
 // fission resources.
 func readSpecs(specDir string) (*FissionResources, error) {
+
+	// make sure spec directory exists before continue
+	if _, err := os.Stat(specDir); os.IsNotExist(err) {
+		fatal(fmt.Sprintf("Spec directory %v does not exist", specDir))
```

Comment:
Whenever add a suggestion for the user to fix the problem, something like:

Spec directory ___ doesn't exist.  Run "fission spec init" to create it.

Reply:
done